dqvm/src/main.lisp

```diff
@@ -74,22 +73,18 @@
     (show-welcome))
 
   (when execute
-    ;; Do all file handling here, not in the node code.
     (when (master-node-p)
-      (setf *program* (safely-read-quil execute)))
-
-    (assert (and (plusp +worker-count+)
-                 (power-of-two-p +worker-count+))
-            ()
-            "There has to be a positive power-of-two ~
+      (assert (power-of-two-p +worker-count+)
```

Comment:
I've removed the `plusp` check here because mpiexec will always pass a positive value `MPI_Comm_size`. It bails out with an error if the passed number is negative and uses all available processors if the number is zero.

Reply:
Honestly, the `plusp` check should probably happen inside `power-of-two-p`. It already checks that it's argument is an `unsigned-byte`, but `(power-of-two-p 0)` currently returns `t`. I guess is does not matter in this case, so maybe it's a feature, not a bug.

Reply:
Fair enough.